packages/core/src/index.ts

```diff
@@ -41,20 +41,28 @@ export class Instance implements Instance {
   }
 
   commit(entities: Array<any>) {
-    for (const entity of entities) {
-      if (entity.$id) {
-        for (const [attr, value] of Object.entries(entity)) {
-          this.log.append([
-            entity.$id,
-            attr,
-            value
-          ])
+    with_aggregate(this.log, () => {
+      for (const entity of entities) {
+        if (entity.$id) {
+          for (const [attr, value] of Object.entries(entity)) {
+            this.log.append([
+              entity.$id,
+              attr,
+              value
+            ])
+          }
         }
       }
-    }
+    })
   }
 }
 
+function with_aggregate(log: Index.List, cb: () => void) {
+  log.aggregate_hooks(true);
```

Comment:
```suggestion
  log.aggregateOps(true);
```

By change, two suggestions:

1. I believe, here you don't really aggregating the hooks, but ops.
2. May we start doing something like `#[wasm_bindgen(js_name = aggregateOps)]`?